src/main/java/com/github/platymemo/alaskanativecraft/AlaskaNativeCraftModels.java

```diff
@@ -0,0 +1,34 @@
+package com.github.platymemo.alaskanativecraft;
```

Comment:
This should probably be in `com.github.platymemo.alaskanativecraft.client.model.entity`

Reply:
Moved in ecffd3e2a7851f9764856886c105aca42aaa0a78